lib/generate-report.js

```diff
@@ -399,6 +408,12 @@ function generateReport(options) {
      * @private
      */
     function _createFeatureIndexPages(suite) {
+
+        // Set custom metadata overview for the feature
+        var featuresMetadataOverview;
```

Comment:
Please write this as below and try to avoid the usage of `var`

`FEATURE_METADATA_OVERVIEW_TEMPLATE = suite.customMetadata ? FEATURE_CUSTOM_METADATA_OVERVIEW_TEMPLATE : FEATURE_METADATA_OVERVIEW_TEMPLATE;`